.github/workflows/release.yaml

```diff
@@ -12,50 +12,194 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 name: Release
 
-on:
-  release:
-    types:
-    - created
+run-name: Releasing ${{ inputs.target_version }} ${{ github.event.inputs.branch }}
+env:
+  REPO: quay.io
+  REPO_NAME: nuclio
+  CACHE_REPO: ghcr.io
+  CACHE_REPO_NAME: ${{ github.repository_owner }}
+  DOCKER_BUILDKIT: 1
 
-  # Run Release on push to development for unstable
+permissions:
+  contents: write
+  packages: write
+
+on:
   push:
     branches:
-    - development
-
+      - development
   workflow_dispatch:
     inputs:
+      version:
+        description: 'The version to release. If not provided, will be calculated.'
+        default: ''
+        type: string
+      bump_version_mode:
+        description: 'Version bump mode'
+        default: 'bump-minor'
+        type: choice
+        options: ['bump-minor', 'bump-major', 'bump-patch']
+      target_version:
+        description: 'Target version (Optional)'
+        required: false
+        default: ''
+      helm_chart_target_version:
+        description: 'Helm chart target version (Optional)'
+        required: false
+        default: ''
+      skip_create_tag_release:
+        description: 'Skip creating GitHub tag & release'
+        required: false
+        default: 'false'
+        type: choice
+        options: ['true', 'false']
+      skip_bump_helm_charts:
+        description: 'Skip bumping helm charts'
+        required: false
+        default: 'false'
+        type: choice
+        options: [ 'true', 'false' ]
+      bump_helm_charts_only:
+        description: 'Bump helm charts only'
+        required: false
+        default: 'false'
+        type: choice
+        options: [ 'true', 'false' ]
```

Comment:
This is not the same as `skip_create_tag_release`, right? in the `unstable` case for example.